docs/portal/applications/adding-and-deploying-applications.md

```diff
@@ -1,514 +0,0 @@
-# Adding and Deploying Applications Using a Wizard
```

Comment:
I notice that you've deleted this page and splitted the contents into 4 separate pages. Please modify the "routes" on this file (docs/manifest.json) as well, otherwise the new menus won't appear on the documentation site. Please let me know if you need help in this.
I notice that there are many links that point to this deleted page, as you can see in my other comments. Please check and fix broken links. Note that I might not have spotted all broken links.

Reply:
When I meet w/Eleazar about getting high-res screen shots, I will ask whether he knows how to do this. If he doesn't, I will get back to you, Neil. Thank you.

Reply:
OK, please make further comment if any updates.

Reply:
Updated hierarchy (routes and paths) to address this issue.